### PR TITLE
New quizzes added

### PR DESCRIPTION
**Pull request by Stephenfre**
# Description
Added quizzes for the heat map, symbiosis chart, and temperature chart. Also changed bubble quiz from "Type In" to "Multiple Choice"

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ *] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ *] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [* ] My code follows the style guidelines of this project
- [ *] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ *] I have made corresponding changes to the documentation
- [* ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [* ] There are no merge conflicts

### CODE REVIEW

File: src/features/quiz/BubblesQuiz.js [hunk 1]
```diff
@@ -1,181 +1,137 @@
...
   const [showResults, setShowResults] = useState(false);
-  const [correct, setCorrect] = useState();
+  const [error, setError] = useState([]);
+
+  const questions = [
+    {
+      id: 1,
+      question: "Which country has the largest number of cases?",
+      answer_a: "Brazil",
+      answer_b: "Russias",
```
Comment (serinamarie) [inline src/features/quiz/BubblesQuiz.js:34]: Don't want to butt in, but would answer_b: "Russias" actually just be "Russia"?

File: src/features/quiz/BubblesQuiz.js [hunk 2]
```diff
@@ -1,181 +1,137 @@
...
+      answer_c: "China",
+      answer_d: "United State of America",
```
Comment (serinamarie) [inline src/features/quiz/BubblesQuiz.js:36]: And would answer_d: "United State of America" be "United States of America" ?

File: src/features/quiz/SymbiosisQuiz.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  const [currentAnswer, setCurrentAnswer] = useState("");
+  const [answers, setAnswers] = useState([]);
+  const [showResults, setShowResults] = useState(false);
+  const [error, setError] = useState([]);
+
+  const questions = [
+    {
+      id: 1,
+      question:
+        "On April 16th which US states contain the top 2 COVID - 19 hot spots?",
```
Comment (serinamarie) [inline src/features/quiz/SymbiosisQuiz.js:33]: Again, not to butt in but to save time, I believe this question should be
"Which animal has 3 relationships with other animals, including both mutualism and commensalism?"

File: src/features/visualizations/migration/pattern/Migration.js [hunk 4]
```diff
@@ -134,8 +134,9 @@ export default withErrorBoundary(() => {
         have become less predictable with the advent of increasing climate
         change, so too have the patterns of bird migrations become more varied.
         Bird observations during the peak of the season have decreased between
-        1970 and today, suggesting that the peak of the season is now earlier
-        and that the populations of various species may be decreasing.
+        1970 and today, suggesting that the peak of the season is now earlier.
+        Similar data suggests that the populations of various species may be
+        decreasing.
```
Comment (serinamarie) [inline src/features/visualizations/migration/pattern/Migration.js:139]: My final act of butting in is merely to request that the full copy for the migration visualization be included, which just means adding this paragraph:

"A density plot allows us to check the distribution of a numeric variable. This is useful when examining a new dataset. A ridgeline plot such as this one lets us look at the distribution of bird observations amongst several groups. In this case, each group is the migration for that year. Each ‘observation’ is the number of sightings for a particular species."

I'll see myself out now.

File: src/features/quiz/SymbiosisQuiz.js (hunk 3 above)
Comment (Stephenfre) [reply src/features/quiz/SymbiosisQuiz.js:33]: This is what happens when I copy and paste too much lol

File: src/features/quiz/Results.js [hunk 5]
```diff
@@ -1,19 +1,19 @@
...
+//           <h1>Correct</h1>
+//         </div>
+//       ) : (
+//         <div className="wrong-answer">
+//           <h1>Failed</h1>
+//         </div>
+//       )}
+//     </>
+//   );
+// }
```
Comment (fwesss) [inline src/features/quiz/Results.js:19]: The commented out code can be deleted. If someone needs it later, they can find it in the git history.

**Pull request merged**